### PR TITLE
Add MySite* values to context interface in 2.0 preview

### PR DESCRIPTION
**Pull request by cloudtx**
Add MySite* values to context interface in 2.0 preview

### CODE REVIEW

**Review by MarcinJuraszek — CHANGES_REQUESTED** [review]
This just adds it to the old context. How is this going to be exposed in the new multi-lavel context structure?

**Review by anmaur — APPROVED** [review]
LGTM!!, Thanks Stephen for making the changes in 2.0

File: packages/teams-js/src/public/app.ts [hunk 1]
```diff
@@ -732,10 +742,12 @@ function transformLegacyContextToAppContext(legacyContext: LegacyContext): app.C
     sharePointSite:
       legacyContext.teamSiteUrl || legacyContext.teamSiteDomain || legacyContext.teamSitePath
```
Comment (MarcinJuraszek) [inline packages/teams-js/src/public/app.ts:743]: should this list include the new properties too?

Comment (MarcinJuraszek) [reply packages/teams-js/src/public/app.ts:743]: as in, is it possible to have `mySiteUrl` present on context with all these being undefined?

Comment (cloudtx) [reply packages/teams-js/src/public/app.ts]: @MarcinJuraszek Yes. I've made this update.

**Review by MarcinJuraszek — APPROVED** [review]

**Pull request merged**